test/generator_test.rb

```diff
@@ -0,0 +1,96 @@
+require "test_helper"
+require "generators/stimulus/stimulus_generator"
+
+module Rails
+  def self.root
+    path = File.expand_path("../tmp", __dir__)
+
+    Pathname.new path
+  end
+end
+
```

Comment:
In order to make the tests pass, rails generators expect a root path. I've monkey-patched the Rails.root method here. 

Is there a better way to set the root path?

Reply:
I think this is fine 👍